src/pages/Cart/Cart.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import OrderItems from './OrderItems/OrderItems';
+import './Cart.scss';
+
+class Cart extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      items: [],
+      totalPrice: 0,
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/item.json')
+      .then(res => res.json())
+      .then(res => {
+        this.setState({
+          items: res.items,
+        });
+      });
+  }
+
+  plusBtn = e => {
+    const { items } = this.state;
+    const changeList = [...items];
+    changeList.map(el => {
+      if (Number(e.target.name) === el.id) {
+        el.quantity = el.quantity + 1;
+      }
+    });
```

Comment:
- 복사를 하고는 계시지만 얕은 복사 한 객체의 내부 요소를 바꾸는 것이라 실제로는 직접 state를 변경하고 있는 상황입니다. 얕은 복사 / 깊은 복사 / 객체 불변성에 대해서 한번 공부해보시면 좋을거 같아요. 일단 위 코드는 아래처럼 바꾸면 좋겠습니다.
- 그리고 el 같은 불명확한 이름보다는 기왕 items라고 해둔 김에 그것으로부터 뽑혀나온 개별 요소들은 Item으로 이름 붙여주면 좋을거 같습니다.
```suggestion
  plusBtn = e => {
    const { items } = this.state;
    
    const changeList = items.map(item => {
      if (Number(e.target.name) === item.id) {
        return { ...item, quantity: el.quantity + 1 }
      } else {
        return item
      }
    });
```